com.woltlab.wcf/package.xml

```diff
@@ -62,4 +62,55 @@
 		
 		<instruction type="objectType" />
 	</instructions>
+	
+	<instructions type="update" fromversion="3.1.10 pl 2">
+		<instruction type="sql">update_5.2_pre_sql_api.sql</instruction>
+		
+		<!-- contains 3 PIP-scripts, \wcf\data\option\OptionEditor, \wcf\system\cache\builder\RoutingCacheBuilder, \wcf\system\database\table\ -->
```

Comment:
What is the rationale behind adding `OptionEditor` and `RoutingCacheBuilder` into it? Neither of those include any changes that are required for the upgrade itself?